lib/krane/cluster_resource_discovery.rb

```diff
@@ -101,6 +99,22 @@ def version_for_kind(versions, kind)
       version_override.fetch(kind, latest)
     end
 
+    def get_resource_api_path(api_versions, resource)
```

Comment:
`def gvk_string(...` ?

I think of the api path as thing you'd pass to `kubectl get --raw ` (though maybe that's just me).

Reply:
`def gvk` also works, though I'm ok with any of these options, really